xilem/src/view/sized_box.rs

```diff
@@ -22,13 +25,15 @@ where
         inner,
         height: None,
         width: None,
+        background: Mutex::default(),
     }
 }
 
 pub struct SizedBox<V> {
     inner: V,
     width: Option<f64>,
     height: Option<f64>,
+    background: Mutex<Option<BackgroundBrush>>,
```

Comment:
I also am not happy with the mutex. But, I've avoided blocking wait anywhere and just used `try_lock`, as I don't think it would ever fail.
(Handled the failing cases though. No panics 😊)